src/Utils/Data/Config/Settings/LastCalculation.java

```diff
@@ -15,28 +15,28 @@
 import Utils.Data.Config.Config;
 
 public class LastCalculation {
-    String baseCur, targetCur, amount, lastFetchTime;
```

Comment:
Temporär??? 
Ich weiß es ist WIP aber ich frag lieber schon mal..

Reply:
ich würde LastFetchTime in der Methode entfernen, da es ein bisschen überflüssig ist. Ich hab die Methode so gemacht, dass es die eingegebenen Daten wieder abruft und daraus erneut eine Rechnung macht. Selbst wenn wir Daten (mit dem Ergebnis) speichern sollten, denke ich nicht, dass FetchTime nötig ist.